codecov_auth/models.py

```diff
@@ -1,2 +1 @@
 from shared.django_apps.codecov_auth.models import *
-from shared.django_apps.codecov_auth.models import _generate_key  # noqa: F401
```

Comment:
Is this safe to remove? I forget specifically why this was here in the first place, maybe for some migrations?

Reply:
searching for this across the code did not yield any results, so 🤷🏻‍♂️

Reply:
Did a quick search and found it here https://github.com/codecov/codecov-api/blob/2f0dbaf5817d46cb5fd5652b1246c3574d8d5b7f/core/migrations/0001_initial.py#L95

So I think by completely removing that migration from API and putting it into Shared we should be able to delete it

Reply:
Indeed, the migration as it exists in shared already has copied this code:
https://github.com/codecov/shared/blob/95c63cd9647a825355db6b75c370ccc32d0b1704/shared/django_apps/core/migrations/0001_initial.py#L13